design/ceph/object/zone-group.md

```diff
@@ -53,17 +54,36 @@ The order in which these resources are created is not important.
 1. The realm named in the `realm` section must be the same as the ceph-object-realm resource the zone group is a part of.
 3. When resource is deleted, zone group are not deleted from the cluster. Zone group deletion must be done through toolboxes.
 
+#### Hostname Configuration
+
+ A list of hostnames to use for accessing the bucket directly like a subdomain in the endpoint. For example if the default service endpoint `rook-ceph-rgw-my-store.rook-ceph.svc:80` added and the object store contains a bucket named `sample`, then the s3 style dns domain would be `http://sample.rook-ceph-rgw-my-store.rook-ceph.svc:80`. More details can be found in [IBM Documentation](https://www.ibm.com/docs/en/storage-ceph/6?topic=configuration-add-wildcard-dns).
```

Comment:
Is there a topic in upstream ceph docs? Let's prefer to use ceph.com links instead of other sites.

Reply:
Jiffin said there are no upstream docs, which matches what I found as well. Though I did find the RH docs to be more helpful to understand the feature than the IBM docs, which are a context-less how-to. I'm pretty sure I linked in my past review to the RH docs. @thotz could you please link there instead?

Reply:
That link points to older ceph version, the new docs https://access.redhat.com/documentation/en-us/red_hat_ceph_storage/6/html/object_gateway_guide/basic-configuration#add-a-wildcard-to-the-dns-rgw is same for both red hat and ceph